### DISCUSSION TITLE
Features moved from Issues to Discussion

### DISCUSSION THREAD

**Discussion by mgogoulos** [mediacms-io/mediacms#659] [Feature Requests]
I've moved almost all of the feature requests from the Issues section to the Discussions, now Issues are mostly Bugs and things that have to be fixed. I realized that the big number of issues can be very intimidating while they are not really issues but rather suggestions and feature requests.

**Comment by KyleMaas** [comment]
For me, I tend to prefer when projects have things filed as issues with "Bug", "Feature Request", etc. tags.  Makes it a little easier to cross-reference between them, browse and search, and reference them in pull requests.  Falls closer in line with my expectations from other projects.

It also makes it very easy to see at a glance that there is an active community - number of filed issues actually makes me feel better about looking at a project because it means it has more eyes on it, especially if I know that a lot of those "issues" are actually feature requests by looking at the tags.  There's no counter at the top of the page for the number of active discussions, so that aspect of the community gets somewhat hidden and makes the project look like it's not very active in comparison to, say, PeerTube where I can look at see 406 issues and know a bunch of people are actively working on it.

When I look at a project, an "issue" for me is not synonymous with "bug".  It has more of a meaning of "someone cared enough to give feedback".

My $0.02.
